### ISSUE TITLE
RotateLoader does not render

### ISSUE THREAD

**Issue by jobbogamer** [davidhu2000/react-spinners#22]
**Do you want to request a feature or report a bug?**
Report a bug.

**If reporting a bug, what version of `react-spinners` are you currently using?**
0.2.5

**What is the current behavior?**
The `RotateLoader` does not render anything on the page and appears as an empty 0 x 0 element.

**If the current behavior is a bug, please provide the steps to reproduce**
```
import React from 'react';
import { RotateLoader } from 'react-spinners';

const MyComponent = () => <RotateLoader />;
```

**Comment by jobbogamer** [comment]
The `RotateLoader` also does not render on the demo site.

**Comment by davidhu2000** [comment]
looks like whatever the issue is, it is affect a lot of loaders, going to revert the commit for now and investigate later.

**Comment by davidhu2000** [comment]
@jobbogamer can you try to use `v0.2.2` and see if you experience the same rendering issues?

**Comment by jobbogamer** [comment]
@davidhu2000 I've just tried it out with v0.2.2 and I see the same behaviour, although I am using React 16.2 and it looks like versions of `react-spinners` before v0.2.3 were depending on React 15?

**Comment by davidhu2000** [comment]
@jobbogamer looks like it might be related to react 16.

I don't think there is anything strictly dependent on the react version, but using react 15 solves the render issue. 

Will continue to investigate this.

**Comment by audrium** [comment]
Same happened to me with the PacmanLoader after migrating to React 16. It doesn't work with version v0.2.2 either. It also does not render on the demo

**Comment by davidhu2000** [comment]
sorry, haven't had much time to work on this lately, but the issue seems to be related to react 16. 

Switching back to React 15 fixed the render issue.

**Comment by davidhu2000** [comment]
This is fixed in `v0.3.2`
